api/v1/helpers.go

```diff
@@ -198,6 +199,8 @@ func MakeVDB() *VerticaDB {
 			ClientServerTLS:   &TLSConfigSpec{Enabled: BoolPtr(true)},
 		},
 	}
+	vdb.Annotations[vmeta.ValidLicenseKeyAnnotation] = "license.dat"
```

Comment:
Is there any specific reason for separating this annotation? Could it be added to the above VerticaDB configuration?